From 969b9bd7ba709f1f997a786b26d8bddb7376e824 Mon Sep 17 00:00:00 2001
From: Silvio Traversaro <silvio@traversaro.it>
Date: Tue, 23 Mar 2021 11:39:53 +0100
Subject: [PATCH] FindODE: try to check first for ODE cmake config

---
 cmake/FindODE.cmake | 13 ++++++++++++-
 1 file changed, 12 insertions(+), 1 deletion(-)

diff --git a/cmake/FindODE.cmake b/cmake/FindODE.cmake
index a447eee175a..d499937cdd9 100644
--- a/cmake/FindODE.cmake
+++ b/cmake/FindODE.cmake
@@ -17,6 +17,18 @@
 # and the following targets:
 #   ODE::ODE
 
+include(FindPackageHandleStandardArgs)
+
+# First try to find ODE via the official cmake config
+find_package(ODE QUIET NO_MODULE)
+mark_as_advanced(ODE_DIR)
+
+if(ODE_FOUND)
+  find_package_handle_standard_args(ODE CONFIG_MODE)
+  return()
+endif()
+
+# Fallback to use pkg-config
 find_package(PkgConfig QUIET)
 
 # Check to see if pkgconfig is installed.
@@ -41,7 +53,6 @@ endif()
 set(ODE_VERSION ${PC_ODE_VERSION})
 
 # Set (NAME)_FOUND if all the variables and the version are satisfied.
-include(FindPackageHandleStandardArgs)
 find_package_handle_standard_args(ODE
     FAIL_MESSAGE  DEFAULT_MSG
     REQUIRED_VARS ODE_INCLUDE_DIRS ODE_LIBRARIES
