From 962eb687d113f8ea92ac011d05b8b95c40c05c44 Mon Sep 17 00:00:00 2001
From: Silvio Traversaro <silvio@traversaro.it>
Date: Fri, 20 May 2022 10:25:21 +0200
Subject: [PATCH] Use octomap also on Windows

---
 cmake/DARTFindDependencies.cmake | 38 ++++++++++++--------------------
 1 file changed, 14 insertions(+), 24 deletions(-)

diff --git a/cmake/DARTFindDependencies.cmake b/cmake/DARTFindDependencies.cmake
index 3cdb6ea8df61..d7f76c42cdf3 100644
--- a/cmake/DARTFindDependencies.cmake
+++ b/cmake/DARTFindDependencies.cmake
@@ -91,33 +91,23 @@ endif()
 
 # octomap
 dart_find_package(octomap)
-if(MSVC)
-  # Supporting Octomap on Windows is disabled for the following issue:
-  # https://github.com/OctoMap/octomap/pull/213
-  message(WARNING "Octomap ${octomap_VERSION} is found, but Octomap "
-      "is not supported on Windows until "
-      "'https://github.com/OctoMap/octomap/pull/213' "
-      "is resolved.")
-  set(HAVE_OCTOMAP FALSE CACHE BOOL "Check if octomap found." FORCE)
-else()
-  if(OCTOMAP_FOUND OR octomap_FOUND)
-    if(NOT DEFINED octomap_VERSION)
-      set(HAVE_OCTOMAP FALSE CACHE BOOL "Check if octomap found." FORCE)
-      message(WARNING "Looking for octomap - octomap_VERSION is not defined, "
-          "please install octomap with version information"
-      )
-    else()
-      set(HAVE_OCTOMAP TRUE CACHE BOOL "Check if octomap found." FORCE)
-      if(DART_VERBOSE)
-        message(STATUS "Looking for octomap - version ${octomap_VERSION} found")
-      endif()
-    endif()
-  else()
+if(OCTOMAP_FOUND OR octomap_FOUND)
+  if(NOT DEFINED octomap_VERSION)
     set(HAVE_OCTOMAP FALSE CACHE BOOL "Check if octomap found." FORCE)
-    message(WARNING "Looking for octomap - NOT found, to use VoxelGridShape, "
-        "please install octomap"
+    message(WARNING "Looking for octomap - octomap_VERSION is not defined, "
+        "please install octomap with version information"
     )
+  else()
+    set(HAVE_OCTOMAP TRUE CACHE BOOL "Check if octomap found." FORCE)
+    if(DART_VERBOSE)
+      message(STATUS "Looking for octomap - version ${octomap_VERSION} found")
+    endif()
   endif()
+else()
+  set(HAVE_OCTOMAP FALSE CACHE BOOL "Check if octomap found." FORCE)
+  message(WARNING "Looking for octomap - NOT found, to use VoxelGridShape, "
+      "please install octomap"
+  )
 endif()
 
 #=======================
