From 4d9052429b3133f08d6ebee6d8af5932bc670a24 Mon Sep 17 00:00:00 2001
From: Silvio Traversaro <silvio@traversaro.it>
Date: Sun, 10 Oct 2021 11:57:32 +0200
Subject: [PATCH] Update CMakeLists.txt

---
 dart/external/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dart/external/CMakeLists.txt b/dart/external/CMakeLists.txt
index aa2fa0a2121a..0d04e5b43df2 100644
--- a/dart/external/CMakeLists.txt
+++ b/dart/external/CMakeLists.txt
@@ -1,5 +1,5 @@
 add_subdirectory(convhull_3d)
-add_subdirectory(imgui)
+#add_subdirectory(imgui)
 add_subdirectory(ikfast)
 add_subdirectory(lodepng)
 add_subdirectory(odelcpsolver)
 
