From 48731e9e020e7c9d80fa55564240b43d2832a26e Mon Sep 17 00:00:00 2001
From: Silvio Traversaro <silvio@traversaro.it>
Date: Tue, 23 Mar 2021 12:10:09 +0100
Subject: [PATCH] Make ode required

---
 cmake/DARTFindODE.cmake | 10 +---------
 1 file changed, 1 insertion(+), 9 deletions(-)

diff --git a/cmake/DARTFindODE.cmake b/cmake/DARTFindODE.cmake
index 9833f5acce8..0c8bf720d4c 100644
--- a/cmake/DARTFindODE.cmake
+++ b/cmake/DARTFindODE.cmake
@@ -6,12 +6,4 @@
 #
 # This file is provided under the "BSD-style" License
 
-find_package(ODE 0.13 QUIET MODULE)
-
-if(ODE_FOUND AND NOT TARGET ODE::ODE)
-  add_library(ODE::ODE INTERFACE IMPORTED)
-  set_target_properties(ODE::ODE PROPERTIES
-    INTERFACE_INCLUDE_DIRECTORIES "${ODE_INCLUDE_DIRS}"
-    INTERFACE_LINK_LIBRARIES "${ODE_LIBRARIES}"
-  )
-endif()
+find_package(ODE REQUIRED CONFIG)
