diff --git a/python/dartpy/CMakeLists.txt b/python/dartpy/CMakeLists.txt
--- a/python/dartpy/CMakeLists.txt
+++ b/python/dartpy/CMakeLists.txt
@@ -19,6 +19,9 @@
 
 file(GLOB_RECURSE dartpy_headers "*.h" "*.hpp")
 file(GLOB_RECURSE dartpy_sources "*.cpp")
+if(NOT DART_BUILD_GUI_OSG)
+  list(FILTER dartpy_sources EXCLUDE REGEX "/gui/osg/")
+endif()
 
 # Python binding module name
 set(pybind_module dartpy)
@@ -47,8 +50,10 @@
   dart-utils
   dart-utils-urdf
   dart-gui
-  dart-gui-osg
 )
+if(DART_BUILD_GUI_OSG)
+  list(APPEND dartpy_libraries dart-gui-osg)
+endif()
 foreach(dartpy_library ${dartpy_libraries})
   if(NOT TARGET ${dartpy_library})
     message(FATAL_ERROR "Cannot find the library ${dartpy_library}. Please check the configuration of DART.")
@@ -70,6 +75,7 @@
 
 target_compile_definitions(${pybind_module}
   PRIVATE DARTPY_VERSION_INFO="${DARTPY_VERSION_INFO}"
+  PRIVATE DART_BUILD_GUI_OSG=$<BOOL:${DART_BUILD_GUI_OSG}>
 )
 
 # Remove debug postfix for dartpy
diff --git a/python/dartpy/gui/module.cpp b/python/dartpy/gui/module.cpp
--- a/python/dartpy/gui/module.cpp
+++ b/python/dartpy/gui/module.cpp
@@ -37,13 +37,17 @@
 namespace dart {
 namespace python {
 
+#if DART_BUILD_GUI_OSG
 void dart_gui_osg(py::module& m);
+#endif
 
 void dart_gui(py::module& m)
 {
   auto sm = m.def_submodule("gui");
 
+#if DART_BUILD_GUI_OSG
   dart_gui_osg(sm);
+#endif
 }
 
 } // namespace python
