diff --git a/CMakeLists.txt b/CMakeLists.txt
index fc5249444a5c..774cd1a83c1b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -390,7 +390,7 @@ if(TARGET dart)
 
   # Add example subdirectories and an "examples" target.
   if(MSVC)
-    add_subdirectory(examples)
+    # add_subdirectory(examples)
   else()
     add_subdirectory(examples EXCLUDE_FROM_ALL)
     get_property(examples GLOBAL PROPERTY DART_EXAMPLES)
@@ -410,7 +410,7 @@ if(TARGET dart)
 
   # Add a "tutorials" target to build tutorials.
   if(MSVC)
-    add_subdirectory(tutorials)
+    # add_subdirectory(tutorials)
   else()
     add_subdirectory(tutorials EXCLUDE_FROM_ALL)
     get_property(tutorials GLOBAL PROPERTY DART_TUTORIALS)
